core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -310,8 +310,8 @@
   USER_LOCAL_WRITER_PACKET_SIZE_BYTES(Name.USER_LOCAL_WRITER_PACKET_SIZE_BYTES, "64KB"),
   USER_PACKET_STREAMING_ENABLED(Name.USER_PACKET_STREAMING_ENABLED, false),
   USER_RPC_RETRY_BASE_SLEEP_MS(Name.USER_RPC_RETRY_BASE_SLEEP_MS, 50),
-  USER_RPC_RETRY_MAX_SLEEP_MS(Name.USER_RPC_RETRY_MAX_SLEEP_MS, 10000),
-  USER_RPC_RETRY_MAX_NUM_RETRY(Name.USER_RPC_RETRY_MAX_NUM_RETRY, 30),
+  USER_RPC_RETRY_MAX_SLEEP_MS(Name.USER_RPC_RETRY_MAX_SLEEP_MS, 5000),
+  USER_RPC_RETRY_MAX_NUM_RETRY(Name.USER_RPC_RETRY_MAX_NUM_RETRY, 20),
```

Comment:
the maximum wait time is now ~70 seconds (71.35 to be precise) ... there is a trade-off between the client waiting for too long to return an error to the application (looks as if the client is hanging) vs. the client client not waiting long enough for a crashed process to restart or the master failover to happen

establishing a contract between the clients and serves such that the clients can tolerate any service disruption of less than 60 seconds seems a reasonable tradeoff to me ... what do you think?